### PR TITLE
test: kubeops, argo-cluster-generation-from-rancher-capi, consul

### PR DESCRIPTION
**Pull request by mintbomb27**
## Related Issue(s)
#546 

## Description
Contains test for:
* consul/enforce-min-tls-version
* kubeops/config-syncer-secret-generation-from-rancher-capi
* argo/argo-cluster-generation-from-rancher-capi

## Checklist

- [x] I have read the [policy contribution guidelines](https://github.com/kyverno/policies/blob/main/README.md#contribution).
- [x] I have added test manifests and resources covering both positive and negative tests that prove this policy works as intended.
- [x] I have added the artifacthub-pkg.yml file and have verified it is complete and correct.

### CODE REVIEW

File: argo/argo-cluster-generation-from-rancher-capi/01-policy.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+apply:
+- argo-cluster-generation-from-rancher-capi.yaml
```
Comment (chipzoller) [inline argo/argo-cluster-generation-from-rancher-capi/01-policy.yaml:4]: As this is another "generate existing" policy, we should probably also have a step here which creates a Cluster resource first and then installs the policy. The Secret is expected to be generated both after the policy has been installed AND after a second Cluster resource has been created.

Comment (chipzoller) [reply argo/argo-cluster-generation-from-rancher-capi/01-policy.yaml:4]: I also don't see a `00` step to assert the CRD here.

File: kubeops/config-syncer-secret-generation-from-rancher-capi/01-policy.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+apply:
+- config-syncer-secret-generation-from-rancher-capi.yaml
```
Comment (chipzoller) [inline kubeops/config-syncer-secret-generation-from-rancher-capi/01-policy.yaml:4]: Ditto here with respect to comments made on the Argo policy above.

**Review by chipzoller — CHANGES_REQUESTED** [review]

File: argo/argo-cluster-generation-from-rancher-capi/01-policy.yaml (hunk 1 above)
Comment (mintbomb27) [reply argo/argo-cluster-generation-from-rancher-capi/01-policy.yaml]: I've added.

File: kubeops/config-syncer-secret-generation-from-rancher-capi/01-policy.yaml (hunk 2 above)
Comment (mintbomb27) [reply kubeops/config-syncer-secret-generation-from-rancher-capi/01-policy.yaml]: Done!

**Review by chipzoller — APPROVED** [review]

**Pull request merged**